src/views/splash/splash.jsx

```diff
@@ -344,25 +343,39 @@ var Splash = injectIntl(React.createClass({
         return (
             <div className="splash">
                 {this.shouldShowEmailConfirmation() ? [
-                    <DropdownBanner key="confirmedEmail"
-                            className="warning"
-                            onRequestDismiss={this.handleDismiss.bind(this, 'confirmed_email')}>
+                    <DropdownBanner
+                        key="confirmedEmail"
+                        className="warning"
+                        onRequestDismiss={this.handleDismiss.bind(this, 'confirmed_email')}
+                    >
                         <a href="#" onClick={this.showEmailConfirmationModal}>Confirm your email</a>
                         {' '}to enable sharing.{' '}
                         <a href="/info/faq/#accounts">Having trouble?</a>
                     </DropdownBanner>,
-                    <Modal key="emailConfirmationModal"
-                           isOpen={this.state.emailConfirmationModalOpen}
-                           onRequestClose={this.hideEmailConfirmationModal}
-                           style={{content: emailConfirmationStyle}}>
-                        <iframe ref="emailConfirmationiFrame"
-                                src="/accounts/email_resend_standalone/"
-                                {...omit(emailConfirmationStyle, 'padding')} />
-                    </Modal>
+                    <IframeModal
+                        isOpen={this.state.emailConfirmationModalOpen}
+                        onRequestClose={this.hideEmailConfirmationModal}
+                        className="mod-confirmation"
+                        componentRef="emailConfirmationiFrame"
+                        src="/accounts/email_resend_standalone/"
+                    />
                 ] : []}
                 {this.props.permissions.educator ? [
                     <TeacherBanner key="teacherbanner" messages={messages} />
                 ] : []}
+                <TTTModal
+                    title="Make it Fly"
+                    description="Animate the Scratch Cat, the Powerpuff Girls, or even a taco!"
+                    tutorialLoc="/"
+                    activityLoc="/pdfs/cards/Scratch2Cards.pdf"
+                    guideLoc="/pdfs/cards/Scratch2Cards.pdf"
+                    bannerUrl="/images/ttt/make-it-fly-banner.jpg"
+                    thumbUrl=""
+                    modalProps={{
+                        isOpen: true,
+                        ref: 'tttModal'
```

Comment:
This wasn't supposed to be committed – tower was supposed to exclude this, so I'll remove the whole thing